dali/python/nvidia/dali/plugin/numba/experimental.py

```diff
@@ -296,3 +311,145 @@ def run_cfunc(out_ptr, out_shapes_ptr, out_ndims_ptr, num_outs, in_ptr, in_shape
         self._preserve = True
 
 ops._wrap_op(NumbaFunction, "fn.experimental", "nvidia.dali.plugin.numba")
+
+
+class NumbaFunctionCuda(metaclass=ops._DaliOperatorMeta):
```

Comment:
Maybe you can share a common part with the `NumbaFunction`. I see a lot of repetition here.

Reply:
Done